tests/issue-442-build-config-name/src/test/java/io/dekorate/examples/kuberentes/Issue442BuildConfigNameIT.java

```diff
@@ -0,0 +1,15 @@
+package io.dekorate.examples.kuberentes;
```

Comment:
It seems like the package is misspelled. Not really important of course, just noting it so you can take care of it if you like

Reply:
Fixed.

Reply:
Thanks